x-pack/plugin/rollup/src/main/java/org/elasticsearch/xpack/rollup/action/TransportRollupSearchAction.java

```diff
@@ -288,17 +290,16 @@ static QueryBuilder rewriteQuery(QueryBuilder builder, Set<RollupJobCaps> jobCap
             String fieldName = range.fieldName();
             // Many range queries don't include the timezone because the default is UTC, but the query
             // builder will return null so we need to set it here
-            String timeZone = range.timeZone() == null ? DateTimeZone.UTC.toString() : range.timeZone();
+            String timeZone = range.timeZone() == null ? DateHistogramGroupConfig.DEFAULT_ZONEID_TIMEZONE.toString() : range.timeZone();
```

Comment:
maybe it would better to keep the ZoneId type as long as it is not needed to be converted?
So here and then passing to rewriteFieldName where it would be converted to string?